esmvaltool/utils/draft_release_notes.py

```diff
@@ -41,44 +41,44 @@
 }
 
 PREVIOUS_RELEASE = {
-    'esmvalcore': datetime.datetime(2022, 10, 13, 18),
+    'esmvalcore': datetime.datetime(2023, 5, 19, 16),
     'esmvaltool': datetime.datetime(2022, 10, 28, 18),
 }
 
-
 LABELS = {
     'esmvalcore': (
+        'backwards incompatible change',  # important, keep at the top
+        'deprecated feature',  # important, keep at the top
+        'bug',  # important, keep at the top
```

Comment:
maybe add the `orphan` label suggested by @zklaus too?

Reply:
Nah, I think rather not: The `orphan` label is needed so we decide to pick up abandoned PRs or close them before they are merged. If they show up in the release notes, they have been merged and so the `orphan` label has already served its purpose.

Reply:
ah good point, they'd be `adopted orphan` here then :rofl: